pkg/transform/gorm/models/chainmessage.go

```diff
@@ -16,6 +16,7 @@ type Message struct {
 	GasPremium types.DbToken
 	Method     uint64
 	Params     []byte
+	Signature  string `gorm:"jsonb"`
```

Comment:
Perhaps this should be raw bytes instead of json. The message signature type can be marshalled to CBOR before persisting to this field: https://github.com/filecoin-project/go-state-types/blob/master/crypto/signature.go#L51

Reply:
Oh that's a good idea. Transformations on the basic Filecoin types can always be done later.